bigquery/bigquery_create_connection_cloud_resource/main.tf

```
@@ -0,0 +1,26 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START bigquery_create_connection_cloud_resource]
+## This creates a cloud resource connection.
+## Note: The cloud resource nested object has only one output field - serviceAccountId.
+resource "google_bigquery_connection" "connection" {
```

The default resource name should be "default"
```suggestion
resource "google_bigquery_connection" "default" {
```